processor/processor.go

```diff
@@ -817,8 +819,14 @@ func (proc *Handle) backendConfigSubscriber(ctx context.Context) {
 					for j := range source.Destinations {
 						destination := &source.Destinations[j]
 						destinationIDtoTypeMap[destination.ID] = destination.DestinationDefinition.Name
-						oneTrustConsentCategoriesMap[destination.ID] = oneTrustConsentCategories(destination)
-						ketchConsentCategoriesMap[destination.ID] = ketchConsentCategories(destination)
+						oneTrustConsentCategoriesMap[destination.ID] = GetOneTrustConsentCategories(destination)
+						ketchConsentCategoriesMap[destination.ID] = GetKetchConsentCategories(destination)
+
+						err := error(nil)
```

Comment:
this is causing a linter issue
we can declare it up there with other `var`s

Reply:
Thank you. I've addressed it.